src/pages/Login/ButtonLoginSignUp.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+import './ButtonLoginSignUp.scss';
+import Login from './Login.js';
+import Signup from './Signup.js';
+
+class ButtonLoginSignUp extends Component {
+  constructor() {
+    super();
+    this.state = {
+      loginModalStatus: false,
+      signUpModalStatus: false,
+    };
+  }
+  loginOpen = () => {
+    this.setState({ loginModalStatus: true });
+  };
+  loginClose = () => {
+    this.setState({ loginModalStatus: false });
+  };
```

Comment:
```suggestion
  handleLoginModal = () => {
    this.setState({ loginModalStatus: !loginModalStatus });
  };
```
이렇게 하나의 함수로 합쳐줄 수 있겠네요!

Reply:
네네 근데 그렇게 합치니까 기능이 하나두개씩 계속 문제가 생기거나 안되더라구요... ㅠㅠ

Reply:
loginModalStauts가 항상 true인값이 필요해서 loginOpen과 loginClose 함수를 구분했는데.. 다른 방법이있을까요?